build.gradle

```diff
@@ -47,6 +47,9 @@ allprojects {
         tasks.withType(JavaCompile) {
             options.compilerArgs += ['-Werror', '-Xlint:deprecation']
             options.errorprone {
+                // errorprone must be temporarily disabled in order to bump the errorprone dependency
+                // to a version without `ErrorProneFlags.getList(String)`.
+                enabled = false
```

Comment:
I've verified that this can be safely removed after the meta-baseline-upgrade by compiling the project without this block using a `publishToMavenLocal`.